tests/integration/test_dataapi.py

```diff
@@ -78,6 +80,23 @@ def test_paging(self):
         assert len(results) == 2
         assert results == {'item_2', 'item_3'}
 
+    def test_real_request_with_special_characters(self):
+        """Make a request against Data Api to ensure the SDK works"""
+        client = DataApi()
+
+        # Change the name to one containing special characters to ensure
+        # signature creation works with special characters in the request
+        items_request['name'] = 'тест'
+
+        res = client.request(self.__build_base_url() + items_endpoint, security, consumer_secret, items_request,
+                             action)
+        returned_json = res.json()
+
+        assert len(returned_json['data']) > 0
```

Comment:
I think this test code is starting to be duplicated a fair bit. Soon might be the time to DRY it. Not for now, though.

Reply:
Agreed. I will create a ticket for Groot as any familiarity with the SDK's is valuable at this stage.